drafts/prometheus-alertmanager-rhocp-open-liberty.adoc

```diff
@@ -0,0 +1,207 @@
++
+. Change the labels to match the `ruleSelector` in your Prometheus. For example, if you used the `prometheus.yaml` in the Kabanero guide, your Prometheus YAML would contain:
++
+```
+ruleSelector:
+    matchLabels:
+      prometheus: k8s
+      role: prometheus-rulefiles
+```
++
+So your PrometheusRule labels must also match:
++
+```
+labels:
+    prometheus: k8s
+    role: prometheus-rulefiles
+```
++
+. Navigate to your Prometheus route and click on Status > Rules to verify that they've been loaded in properly.
++
+image::/img/blog/prometheusAM_rhocp_promUI_rules.png[Prometheus Rules on UI, align="left"]
++
+. To check that your alerts are firing, click on Alerts. Since we did not remove the `./example.rules` rule group, you should already see at least one alert firing. To ensure that your custom Liberty rules are also working, generate some data or drastically lower the threshold of the PromQL query.
++
+image::/img/blog/prometheusAM_rhocp_promUI_alerts.png[Prometheus Alerts on UI, align="left"]
+
+== Create and configure Prometheus Alertmanager to pass alerts to Slack
+
+In this section, you'll create and configure the Prometheus Alertmanager to send customized messages to your Slack channel based on the alerting rules we set up earlier.
+
+=== Creating and configuring the Prometheus Alertmanager to communicate with Slack
+
+Now that Prometheus is set up with rules for our Liberty server, we can create a Prometheus Alertmanager instance with the ability to connect to Slack.
+
+. Create a file named `alertmanager.yaml` (do not change the name) - take note of the directory, as we will be creating a secret using this file. Note that this file is configured the same way as we did in the previous blog post. We'll start with a simple configuration. Your file will look like this:
++
+```
+global:
+  resolve_timeout: 5m
+  slack_api_url: <your slack webhook api url>
+
+# The root route. This route is used as the default
+# if there are no matches in the child routes.
+route:
+  group_by: [ alertname ]
+  group_wait: 10s
+  group_interval: 10s
+  repeat_interval: 5m
+  receiver: 'default_receiver'
+
+receivers:
+- name: 'default_receiver'
+  slack_configs:
+  - channel: 'prometheus-alertmanager-test'
+    title: "{{ range .Alerts }}{{ .Annotations.summary }}\n{{ end }}"
+    text: "*Description*: {{ .CommonAnnotations.description }}\n*Severity*: {{ .CommonLabels.severity }}"
+```
++
+In this example, we have one route that sends the alert to `default_receiver`. The receiver sends the alert to a Slack channel called `prometheus-alertmanager-test`.
+The `CommonAnnotations` come from the `annotations` you specified in your Prometheus Rule. The text is written using the link:https://golang.org/pkg/text/template/[Go templating] system.
++
+. On the RHOCP web console, go to Operators > Installed Operators > Prometheus Operator > Alertmanager and create a new Alertmanager instance. You do not need to change the default YAML.
```

Comment:
Suggested update:
"On the RHOCP web console, **click on** Operators > Installed Operators > Prometheus Operator > Alertmanager and create an **`Alertmanager`** instance. You do not need to change the default YAML."